CHANGELOG.md

```diff
@@ -1,30 +1,57 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format is based on [Common Changelog](https://common-changelog.org).\
-Unlike Common Changelog, the `unreleased` section of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) is preserved with the addition of a tag to specify which type of release should be published and to foster discussions about it inside pull requests. This tag should be one of the names mandated by SemVer, within brackets: `[patch]`, `[minor]` or `[major]`. For example: `## Unreleased [minor]`.
+## Unreleased [patch]
 
-## Unreleased
+_Full changeset and discussions: [#981](https://github.com/ambanum/OpenTermsArchive/pull/981)._
+
+### Added
+
+- Specification of changelog format
+
+### Changed
+
+- Specification of changelog format is now in CONTRIBUTING
 
 ## 0.18.1 - 2022-12-12
+
+_Full changeset and discussions: [#979](https://github.com/ambanum/OpenTermsArchive/pull/979)._
+
 ### Fixed
+
 - Add missing configuration for production
 
 ## 0.18.0 - 2022-12-12
+
+_Full changeset and discussions: [#976](https://github.com/ambanum/OpenTermsArchive/pull/976)._
+
 ### Added
-- Load both the configurations defined in this module and the configurations defined in the module which use it as dependency.
+
```

Comment:
Currently the tool we use remove blank lines around titles and lists. So if we make this change we will have inconsistency in changelog. I opened [an issue](https://github.com/oscarotero/keep-a-changelog/issues/28) on the tool, let's wait for a response. Otherwise, we have to fork the tool and make the change.